samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -11,28 +11,60 @@ CONFIG_NET_IPV4=y
 CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETS_OFFLOAD=y
 
-# LwM2M
-CONFIG_LWM2M_CLIENT_UTILS=y
-CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL_DBG=y
+# These two settings are needed due to a error in zephyrs lwm2m implementation.
+# The error occurs when a multi instance resource is set to have a maximum of one instance.
+# When the fix for this error is integrated, which it presumably will be in NCS 1.7,
+# these settings can be safely removed. See zephyr PR #36657 for details.
+CONFIG_NET_IF_MAX_IPV4_COUNT=2
+CONFIG_LWM2M_CONN_MON_APN_MAX=2
+
+# Sensors
+CONFIG_I2C=y
+CONFIG_SPI=y
+CONFIG_SENSOR=y
+
+# UI
+CONFIG_UI_LOG_LEVEL_DBG=y
+
+# LwM2M and IPSO
 CONFIG_LWM2M=y
+CONFIG_LWM2M_COAP_BLOCK_SIZE=512
+CONFIG_LWM2M_ENGINE_MAX_OBSERVER=15
+CONFIG_LWM2M_ENGINE_MAX_MESSAGES=15
+CONFIG_LWM2M_ENGINE_MAX_PENDING=15
+CONFIG_LWM2M_ENGINE_MAX_REPLIES=15
+CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME=60
```

Comment:
Actually, in my network 60 s is enough to invalidate the DTLS session, which results in retransmissions an re-regsitration :/ Perhaps we should restore the default 30 seconds to improve user experience?

Reply:
Set default to 30